app/views/layout.html

```diff
@@ -18,4 +18,5 @@
 
 {% block body_end %}
   {% include "includes/scripts.html" %}
+  <!-- GOV.UK Prototype kit {{releaseVersion}} -->
```

Comment:
But not all visitors, tech savvy visitors, others won't be looking at the source code